presto-product-tests/README.md

```diff
@@ -107,7 +107,7 @@ groups run the following command:
 
 ```
 ./mvnw install -DskipTests
-presto-product-tests/bin/run_on_docker.sh <profile> -x quarantine,big_query,mysql_connector,postgresql_connector,profile_specific_tests
```

Comment:
We were recommending that they be excluded before because we did not have docker containers, and thus running those tests required setting up MySQL and PostgreSQL manually.  Now that we have docker containers for those databases, running the tests is quick and easy so there is no reason to exclude them.